File: easy/N-Mod-M/N-Mod-M.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShoModulus(line);
        }
    }
    
    public static void ShoModulus(String line){
            
            int firstNum = Integer.valueOf(line.substring(0,line.indexOf(",")));
            int secondNum = Integer.valueOf(line.substring(line.indexOf(",")+1));
            int num = firstNum / secondNum;
            System.out.println(firstNum - num*secondNum);
    }
}

File: easy/N-Mod-M/N Mod M.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
                ShoModulus(line);
            // do something with line
        }
    }
    
    public static void ShoModulus(string line){
        int firstNum = Convert.ToInt32(line.Substring(0,line.IndexOf(",")));
        int secondNum = Convert.ToInt32(line.Substring(line.IndexOf(",")+1));
        int num = firstNum / secondNum;
        Console.WriteLine(firstNum - num*secondNum);
    }
}
